src/web_app_skeleton/config/env.cr

```diff
@@ -1,20 +1,29 @@
-module Lucky::Env
-  extend self
+# Lucky manages environments using `LuckyEnv`. By default, development,
```

Comment:
Can we add a link to the lucky_env github repo?

Reply:
I adjusted the documentation in https://github.com/luckyframework/lucky_cli/pull/655/commits/6acd9028931e5f2df70d80d7519e329106b9fd64. However, instead of linking to the GitHub repository, I link to the configuration section of the Lucky documentation. I thought that would be more helpful. What do you think? 🙂